modules/debts_api/lib/debts_api/v0/one_debt_letter_service.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module DebtsApi
+  class V0::OneDebtLetterService
+    def initialize(user)
+      @user = user
+    end
+
+    def get_pdf
+      # copays = vbs_service.get_copays
+      # debts = dmc_service.get_debts
+
+      Prawn::Document.new(page_size: 'LETTER') do |pdf|
+        add_and_format_logo(pdf)
+      end.render
+    end
+
+    private
+
+    def add_and_format_logo(pdf)
+      logo_path = Rails.root.join('modules', 'debts_api', 'app', 'assets', 'images', 'va_logo.png')
+      pdf.image logo_path, at: [(pdf.bounds.width / 2) - (250 / 2), pdf.cursor], width: 250
+    end
+
+    # def vbs_service
```

Comment:
Is there a reason this commented out code is still in the PR?

Reply:
Hey @rjohnson2011, no good reason really. 

We'll be adding that in a later PR, I removed the comments.

Thank you!